### PR TITLE
MET-4357 fix resource leak

### PR DESCRIPTION
**Pull request by jeortizquan**
created one instance of MediaExtractor, RdfSerializer, RdfDeserializer

### CODE REVIEW

File: src/main/java/eu/europeana/metis/sandbox/service/workflow/MediaProcessingServiceImpl.java [hunk 1]
```diff
@@ -22,75 +19,93 @@
...
+  private final RdfDeserializer rdfDeserializer;
 
   public MediaProcessingServiceImpl(
-      RdfConverterFactory converterFactory,
-      MediaProcessorFactory processorFactory,
-      ThumbnailStoreService thumbnailStoreService) {
-    this.converterFactory = converterFactory;
-    this.processorFactory = processorFactory;
+      ThumbnailStoreService thumbnailStoreService,
+      @Qualifier("rdfSerializer")
```
Comment (stzanakis) [inline src/main/java/eu/europeana/metis/sandbox/service/workflow/MediaProcessingServiceImpl.java:42]: The qualifiers are probably not required since they are exactly identical to the name of the field.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,6 +6,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      - name: Setup ImageMagick
```
Comment (stzanakis) [inline .github/workflows/ci.yml:9]: Is this change required for the unit tests?

Comment (jeortizquan) [reply .github/workflows/ci.yml:9]: Correct. for the context load Integration test. If not the build fails. It will need to setup this as well in the jenkins build.

File: src/main/java/eu/europeana/metis/sandbox/service/workflow/MediaProcessingServiceImpl.java (hunk 1 above)
Comment (jeortizquan) [reply src/main/java/eu/europeana/metis/sandbox/service/workflow/MediaProcessingServiceImpl.java]: yes. correct. removed. done!
